### ISSUE TITLE
Search Help Text Overflow 

### ISSUE THREAD

**Issue by omargawdat** [unfoldadmin/django-unfold#628]
Unfold: 0.33.0
Django: 5
Browser: Latest Chrome
---- 
## Bug: 
search help text can't be displayed all within the search box, so i can't read the whole text. 

## Illustration Image: 
<img width="1077" alt="image" src="https://github.com/user-attachments/assets/64cf0212-0ccb-43d2-a2b2-262cd0bea83d">

## Code: 
```
    search_fields = ["phone_number", "company__username", "company__phone_number", "first_name", "last_name"]
    search_help_text = _("Search by phone number, company username, company phone number, first name, last name")
```

## Suggestion:
we can show it all when the user hover over it as in the permissions. 
<img width="421" alt="image" src="https://github.com/user-attachments/assets/354b6058-52ab-4df0-971a-ee2bd24a036f">

**Comment by asfawayalk** [comment]
Good point also when the search bar is shown like this
UNFOLD = {
"SIDEBAR": {
        "show_search": True
}}

1. The result text will overflow if it is long and I can not read all of it. (Showing the whole text when hover is a good feature)
2. If the list of the search result is long, it has to be scrollable. Now I can not access the results that are beyond the screen

**Comment by lukasvinclav** [comment]
Fixed in #691
